lib/route.coffee

```diff
@@ -159,11 +159,12 @@ class @Route
     auth = @api.config.auth.user.call(endpointContext)
 
     # Get the user from the database
-    if not auth?.user and auth?.userId and auth?.token
-      userSelector = {}
-      userSelector._id = auth.userId
-      userSelector[@api.config.auth.token] = auth.token
-      auth.user = Meteor.users.findOne userSelector
+    if not auth?.user
+      if auth?.userId and auth?.token
```

Comment:
Why the extra layer of nesting? This would short circuit if any check returns `false`. No real benefit, since we have to check for the `auth.user` again anyway.